packages/sources/ignition-address-list/src/config/index.ts

```diff
@@ -0,0 +1,16 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const config = new AdapterConfig({
+  API_KEY: {
+    description: 'An API key for Data Provider',
+    type: 'string',
+    required: true,
+    sensitive: true,
+  },
+  API_ENDPOINT: {
+    description: 'An API endpoint for Data Provider',
+    type: 'string',
+    sensitive: true,
```

Comment:
nit: is this really sensitive?

Reply:
I think sensitive means case sensitive?

Reply:
I am wrong

->

// Set to true to censor this env var from logs. Default = false